.storybook/config.js

```diff
@@ -1,6 +1,6 @@
 
-import { configure, addDecorator, addParameters } from '@storybook/react';
```

Comment:
are we doing `space-in-brackets` @jennasalau ?

Reply:
god damn it, no i thought I fixed that. Will update PR, I also spelt adapters wrong...

Reply:
lol this made me laugh..

Yeah no we are not Til, same with arrays incase you see it